lib/packetgen/config.rb

```diff
@@ -20,12 +21,12 @@ class Config
     # @return [String] 
     attr_reader :ip6addr
 
-    # Create a configuration object. If +iface+ is not set, get first network interface.
-    # If non exists, use loopback one.
+    # Create a configuration object. If +iface+ is not set then 
+    # it will be attempted to be found automatically. 
     # @param [String,nil] iface
     def initialize(iface=nil)
       if iface.nil?
-        iface = NetworkInterface.interfaces.select { |iface| iface != 'lo' }.first
+        iface = Pcap.lookupdev
         iface = NetworkInterface.interfaces.first if iface.nil?
```

Comment:
Unfortunately, this change breaks default behaviour of `Config#initialize`, which used loopback interface if none was given: `Pcap.lookupdev` does not set look up loopback interfaces.
This breaks CI tests.